<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#vip_month_pushbutton, #vip_quarter_pushbutton, #vip_year_pushbutton {  
    border: 1px solid gray;
	border-radius: 8px;
}

#vip_year_pushbutton:focus
{
	background-color: #ffdddd;
	border: 2px solid #ff1493;
}

#vip_quarter_pushbutton:focus{
	background-color: #ffdddd;
	border: 2px solid #ff1493;
}
#vip_month_pushbutton:focus{
	background-color: #ffdddd;
	border: 2px solid #ff1493;
}

#label_2, #label_4, #label_6
{
	font-size: 20px;  
    font-weight: bold;
}

#label_7 {  
    text-decoration: line-through;  
}

#pushButton{
	border: 1px solid white;
	background-color: #ff1493;
	border-radius: 3px;
}

#payment_button{
	border: 1px solid #4e6ef2;
	border-radius:8px;
	background-color: rgba(78,110,242,0.5);
	color:#ffffff;
}

#back_button{
	border: 1px solid #4e6ef2;
	border-radius:8px;
	background-color: rgba(78,110,242,0.5);
	color:#ffffff;
}

#payment_button:hover{
	border: 1px solid #4e6ef2;
	background-color:#4662d9;
}

#payment_button:pressed{
	border: 3px solid #000000;
	background-color: #4e6ef2;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="top_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {  
    background-image: url(:/vip/starry_sky_background.png);  
    background-repeat: no-repeat;  
    background-position: center;  
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="licence_plate_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>同意《vip服务协议》</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请先同意协议</string>
    </property>
   </widget>
   <widget class="QLabel" name="volitale_money_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="QR_code_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {  
    background-image: url(:/vip/load_QR_code.png);  
    background-repeat: no-repeat;  
    background-position: center;  
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vip_month_pushbutton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>￥25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>135</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>月度VIP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vip_quarter_pushbutton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>135</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>季度VIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>￥68</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vip_year_pushbutton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>136</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>年度VIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>￥168</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>￥233</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7.2折</string>
    </property>
   </widget>
   <widget class="QPushButton" name="payment_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>支付成功</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
